tests/client/core/components/TestPaginate.js

```diff
@@ -177,5 +153,47 @@ describe('<Paginate />', () => {
         );
       });
     });
+
+    describe('when the link is to the current page', () => {
```

Comment:
These call `makeLink()` directly so I think it makes sense to leave them under a `describe('makeLink()')` block.